torch/distributions/pareto.py

```diff
@@ -29,8 +29,8 @@ class Pareto(TransformedDistribution):
 
     def __init__(self, scale, alpha):
         self.scale, self.alpha = broadcast_all(scale, alpha)
-        base_dist = Exponential(alpha)
-        transforms = [ExpTransform(), AffineTransform(loc=0, scale=scale)]
+        base_dist = Exponential(self.alpha)
+        transforms = [ExpTransform(), AffineTransform(loc=0, scale=self.scale)]
```

Comment:
@alicanb pointed out that if `scale` and `alpha` are both scalars, then the 0 here will broadcast everything up to dimension-1. Does it seem reasonable to use `loc=variable(0.0)` here instead, at least until Python numbers become 0-dimensional?

Reply:
`self.scale` is already the output of `broadcast_all` so I think it's safe now

Reply:
but until `WITH_SCALARS=1`, 0 may be a higher-dimensional tensor than `self.scale` and therefore this broadcasting is wrong

Reply:
Won't self.scale be at least 1-d?

Reply:
Oh maybe I misunderstood. Is the following true?
- If `WITH_SCALARS=0` then tensors have at least 1 dim and python numbers have 1 dim
- If `WITH_SCALARS=1` then tensors may have 0 dims, and python numbers have 0 dims

If so then @alicanb is right and this is fine.

Reply:
That's right if by "python numbers have X dim" you mean python numbers get broadcasted to a tensor of X dims in broadcast_all; python numbers themselves don't have dims.